tests/topotests/lib/topotest.py

```diff
@@ -1676,7 +1701,7 @@ def startRouter(self, tgen=None):
             self.run_in_window("vtysh", title="vt-%s" % self.name)
 
         if self.unified_config:
-            self.cmd("vtysh -f /etc/frr/frr.conf")
+            self.cmd("vtysh -b -f /etc/frr/frr.conf")
```

Comment:
```
WARNING: Combinining the -f option with -b or -w is NOT SUPPORTED since its results are inconsistent!
```

Reply:
Actually I'm going to remove the -b/--boot and leave it the way it was.. internally it ends up executing the same code.